app/frontend/src/components/designer/FormViewerMultiUpload.vue

```diff
@@ -319,6 +320,65 @@ export default {
         this.progress = true;
         this.$emit('toggleBlock', true);
         const formHtml = document.getElementById('validateForm');
+        //Add custom validation to the Components those are not covered by FormIO Validation class
+        await Utils.eachComponent(
+          this.formSchema.components,
+          function (component) {
+            //Need to cover Validation parts those are not performed by FormIO validate funciton
+            switch (component.type) {
+              case 'number':
+                component.validate.custom =
+                  "if(component.validate.required === true){valid = _.isNumber(input) ? true : 'Only numbers are allowed in a number field.';} else if(input){valid = _.isNumber(input) ? true : 'Only numbers are allowed in a number field.';}" +
```

Comment:
It's hard to tell from just reading the code, but the `if` and `else if` clauses look to be the same. If so, can this line be simplified to:
```
if(component.validate.required === true || input){valid = _.isNumber(input) ? true : 'Only numbers are allowed in a number field.';} +
```